Source/RapyutaSimulationPlugins/Public/Tools/GhostPlayerPawn.h

```diff
@@ -0,0 +1,25 @@
+/**
+ * @file GhostPlayerPawn.h
+ * @brief BaseComponents which is used when spawning Actor from ROS2 service in #ASimulationState.
+ * @copyright Copyright 2020-2022 Rapyuta Robotics Co., Ltd.
+ */
+
+#pragma once
+#include "CoreMinimal.h"
+#include "GameFramework/DefaultPawn.h"
+
+#include "GhostPlayerPawn.generated.h"
+
+class USphereComponent;
+
+/**
+ * @brief Default player pawn without collision and mesh
+ */
+UCLASS(ClassGroup = (Custom), Blueprintable, meta = (BlueprintSpawnableComponent))
+class RAPYUTASIMULATIONPLUGINS_API AGhostPlayerPawn : public ADefaultPawn
```

Comment:
If `DefaultPawnClass = nullptr` in `ARRNetworkGameMode` works, we'd not need this class I think.
Besides, `RR` prefix also?, other classes in `Tools` folder should be updated as well to have the prefix. This seems to have happened in some PRs before, the meaning is mostly just:
* To quickly differentiate itself from UE or 3rd-plugin classes
* To provide better src uniformity, security & copyright

Reply:
- DefaultPawnClass = nullptr does not work. What I need is DefaultPawn without mesh and collision. DefaultPawn has basic functionalities to be controleld by PlayerController
- will rename.